IdentityCore/src/cache/ios/MSIDKeychainTokenCache.m

```diff
@@ -313,18 +313,15 @@ - (BOOL)removeItemsWithKey:(MSIDTokenCacheKey *)key
     return tokenItems;
 }
 
-- (BOOL)saveWipeInfo:(NSDictionary *)wipeInfo
-                  context:(id<MSIDRequestContext>)context
-               error:(NSError **)error
+- (BOOL)saveWipeInfoWithContext:(id<MSIDRequestContext>)context
```

Comment:
I think we still should provide wipeInfo as parameter here, otherwise it is not possible to test this method properly (we can not test that wipe info was really saved into keychain). In my mind, MSIDtokenCache should provide dictionary with wipe info to save.

Reply:
Per discussion offline, it would be sufficient to create a helper method to do the testing.